client/src/partials/templates/bhFindDocument.tmpl.html

```diff
@@ -0,0 +1,170 @@
+                      <i class="fa fa-download"></i>
+                    </a>
+
+                    <button
+                      ng-click="$ctrl.deleteDocument(doc.uuid)"
+                      title="{{ 'FORM.BUTTONS.DELETE' | translate }}"
+                      type="button"
+                      class="btn btn-sm btn-danger">
+                      <i class="fa fa-trash"></i>
+                    </button>
+                  </div>
+                </div>
+              </li>
+
+            </ul>
+          </div>
+        </div>
+
+        <!-- thumbnail display mode  -->
+        <div ng-switch-when="thumbnail" class="display-thumbnail">
+            <div class="document col-xs-6" ng-click="$ctrl.toggleAction($index)"
+              ng-repeat="doc in $ctrl.session.patientDocuments | filter: $ctrl.searchByName">
+              <div class="crop">
+                <div class="document-icon">
+                  <img ng-show="$ctrl.mimeIcon(doc.mimetype) === 'fa-file-image-o'" class="document-thumbnail" src="{{ doc.link }}" alt="" />
+                  <i ng-hide="$ctrl.mimeIcon(doc.mimetype) === 'fa-file-image-o'" class="fa" ng-class="{{ '$ctrl.mimeIcon(doc.mimetype)' }}"></i>
+                </div>
+              </div>
+              <div class="document-lable">
+                <h4 class="media-heading">{{ doc.label }}</h4>
+
+                <div ng-show="$ctrl.session.showAction && $ctrl.selectedIndex === $index">
+                  <i class="fa fa-file-o"></i>
+                  {{ doc.mimetype }} (~{{ doc.size | bytes }})
+                </div>
+
+                <i class="fa fa-calendar-o"></i>
+                {{ doc.timestamp | date }}, {{ 'FORM.LABELS.BY' | translate }}
+                {{ doc.first + ' ' + doc.last }}
+              </div>
+              <div ng-show="$ctrl.session.showAction && $ctrl.selectedIndex === $index" class="document-actions">
+                <button
+                  title="{{ 'PATIENT_DOCUMENT.OVERVIEW' | translate }}"
+                  type="button"
+                  class="btn btn-sm btn-default">
+                  <i class="fa fa-eye"></i>
+                </button>
+
+                <a
+                  ng-href="/{{ doc.link }}" download="{{ doc.label }}"
+                  title="{{ 'PATIENT_DOCUMENT.DOWNLOAD' | translate }}"
+                  type="button"
+                  class="btn btn-sm btn-default">
+                  <i class="fa fa-download"></i>
+                </a>
+
+                <button
+                  ng-click="$ctrl.deleteDocument(doc.uuid)"
+                  title="{{ 'FORM.BUTTONS.DELETE' | translate }}"
+                  type="button"
+                  class="btn btn-sm btn-danger">
+                  <i class="fa fa-trash"></i>
+                </button>
+              </div>
+            </div>
+        </div>
+      </ng-switch>
+
+      <!-- no result  -->
+      <div ng-if="!$ctrl.session.patientDocuments.length" class="alert alert-warning">
```

Comment:
Instead of a warning, I think this should just be informational text.  It is not a bad thing to be missing documents.  What do you think?